gitea/integration_repositories_user_test.go

```diff
@@ -100,6 +100,18 @@ var _ = Describe("Gitea Provider", func() {
 		Expect(getSpec.Equals(postSpec)).To(BeTrue())
 	})
 
+	It("should return correct repo info when creating a repository with wrong UserLogin", func() {
+		repoName := fmt.Sprintf("test-user-repo-creation-%03d", rand.Intn(1000))
+		repoRef := newUserRepoRef("yadda-yadda-yada", repoName)
```

Comment:
we should change `newUserRepoRef` signature and remove the user as it may lead to wrong assumption.